c10/core/SymFloat.h

```diff
@@ -40,6 +40,9 @@ class C10_API SymFloat {
   SymFloat operator*(const SymFloat&) const;
   SymFloat operator/(const SymFloat&) const;
 
+  // Need guidance on where to put this code
+  SymFloat sqrt() const;
```

Comment:
Have you tried writing something like:


```
namespace std {
  template <>
  SymFloat sqrt<SymFloat>(const SymFloat& self) {
    ...
  }
}
```

Not sure if this would work exactly, but maybe we can get `std::sqrt()` to Just Work on symfloats, so we don't need any code changes in the future

Reply:
Can't seem to match the function template specialization  to a function template. Tried for both std:: and c10::complex